.github/workflows/validate-docs-site.yaml

```diff
@@ -22,6 +22,7 @@ jobs:
       with:
         repository: validmind/installation
         path: site/_source/installation
+        ref: nrichers/sc-11108/documentation-pipeline-for-api-publishing
```

Comment:
TO DO: This branch `ref` needs to be removed before merging. It serves only to provide a preview for [Add admin API to CMVM docs #121](https://github.com/validmind/installation/pull/121)